File: mordisco-api/src/main/java/utn/back/mordiscoapi/model/dto/configuracion/ConfiguracionSistemaResponseDTO.java
package utn.back.mordiscoapi.model.dto.configuracion;

import java.math.BigDecimal;
import java.time.LocalDateTime;

public record ConfiguracionSistemaResponseDTO(
        Long id,
        BigDecimal porcentajeGananciasRestaurante,
        BigDecimal radioMaximoEntrega,
        BigDecimal costoBaseDelivery,
        BigDecimal costoPorKilometro,
        BigDecimal montoMinimoPedido,
        BigDecimal porcentajeGananciasRepartidor,
        LocalDateTime fechaActualizacion,
        String emailUsuarioModificacion
) {}

File: mordisco-front/src/app/shared/models/configuracion/ConfiguracionSistemaResponseDTO.ts
export default interface ConfiguracionSistemaResponseDTO{
        id : number,
        comisionPlataforma : number,
        radioMaximoEntrega : number,
        tiempoMaximoEntrega : number,
        costoBaseDelivery : number,
        costoPorKilometro : number,
        montoMinimoPedido: number,
        porcentajeGananciasRepartidor : number,
        modoMantenimiento : boolean,
        mensajeMantenimiento : string,
        fechaActualizacion : string,
        emailUsuarioModificacion : string
}